src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const startPositionX = 0;
+  const startPositionY = 0;
+  const evacuationPositionX = 1400;
+  const evacuationPositionY = 500;
+
```

Comment:
looks better this way. You can also think of a way of grouping them to a single object
```suggestion
  const initialCoordinates = {
    x: 0,
    y: 0,
  };
  
  const evacuationCoordinates = {
    x: 1400,
    y: 500,
  }
```